lib/msal-browser/docs/device-bound-tokens.md

```diff
@@ -0,0 +1,56 @@
+# Acquiring Device Bound Tokens using Web Account Manager (WAM) on Windows
+
+MSAL.js supports acquiring tokens from the Web Account Manager (WAM) on Windows. These tokens are bound to the device they were acquired on and are not cached in the browser's localStorage or sessionStorage.
+
+## Supported Environment
+
+This feature is currently only supported in the following environment:
+
+- A machine running a Windows build that supports the feature (more to come on this)
+- Chrome and Edge browsers or Teams
+- [Windows Accounts extension](https://chrome.google.com/webstore/detail/windows-accounts/ppnbnpeolgkicgegkbkbjmhlideopiji) is installed if using Chrome or Edge
```

Comment:
Can we also make clear which version of the extension is needed?

Reply:
Is it possible to use anything other than the latest? I believe the Chrome store only lets you download the latest and they auto-update when a new version is published.

Reply:
I think its technically possible to be on an old version, and making this explicit can help mitigate any questions that could come as a result of that.

Reply:
Done